package.json

```diff
@@ -32,7 +32,7 @@
       "orders": "11.0 12.0",
       "orders-storage.settings": "1.0",
       "organizations.organizations": "1.0",
-      "pieces": "3.0",
+      "pieces": "3.1",
```

Comment:
3.0 in itself means support for 3.*

Reply:
I will check with the BE team. They have also updated the version from 3.0 to 3.1

Reply:
The min version `3.1` is required to support new endpoint /orders/pieces-requests/